scripts/SREP/get-sosreport/script.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+## validate input
+if [[ -z "$NODE" ]]
+then
+    echo "Variable node cannot be blank"
+    exit 1
+fi
+
+# Import sftp_upload library
+source /managed-scripts/lib/sftp_upload/lib.sh
+
+# Define expected values
```

Comment:
It's always a good practice to define variables at the top of the script just right below the `set -o pipefail`
```suggestion
set -o pipefail

# Define variables for expected values
DUMP_DIR="${PWD}"
DATE="$(date -u +"%Y%m%dT%H%M")"
SOSREPORT_FILENAME="sosreport-${NODE}.tar.xz"
SOSREPORT_FILEPATH="${DUMP_DIR}/${SOSREPORT_FILENAME}"
NAMESPACE="default" # Can be parameterized if needed ?
```